packages/jsii-pacmak/lib/targets/java.ts

```diff
@@ -1580,6 +1583,16 @@ class JavaGenerator extends Generator {
     return this.toJavaType({ fqn: cls.base });
   }
 
+  private toDecoratedJavaType(optionalValue: spec.OptionalValue): string {
+    const nakedType = this.toJavaType(optionalValue.type);
+    return `${optionalValue.optional ? ANN_NULLABLE : ANN_NOT_NULL} ${nakedType}`;
+  }
+
+  private toDecoratedJavaTypes(optionalValue: spec.OptionalValue): string[] {
+    return this.toJavaTypes(optionalValue.type).map(nakedType =>
+      `${optionalValue.optional ? ANN_NULLABLE : ANN_NOT_NULL} ${nakedType}`);
```

Comment:
can you reuse `toDecoratedJavaType` or otherwise find a way to avoid the duplicated code?